### DISCUSSION TITLE
pulsar docker image  pulsar 3.0.0

### DISCUSSION THREAD

**Discussion by slowpao** [apache/pulsar#20896] [Q&A] [answered]
Docker image apachepulsar/pulsar-all: 3.0.0 Pulsar 2.11.0 jar package used in 3.0.0, are there any plans for future updates to the Pulsar 3 image?

**Comment by RobertIndie** [comment, marked as answer]
This is a known issue: https://github.com/apache/pulsar/issues/20420
The Pulsar 3.0.1 will be released to fix it. It's currently in the voting phase: https://lists.apache.org/thread/rd7483vwnbbrh3xmm82yvzd9lv3wznny
